stories/features/shields/components/controls/scriptsControl.tsx

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Feature-specific components
+import {
+  BlockedInfoRow,
+  BlockedInfoRowData,
+  ArrowDownIcon,
+  BlockedInfoRowStats,
+  BlockedInfoRowText,
+  Toggle
+} from '../../../../../src/features/shields'
+
+// Group Components
+import DynamicList from '../list/dynamic'
+
+// Fake data
+import { getLocale } from '../../fakeLocale'
+import data from '../../fakeData'
+
+interface Props {
+  favicon: string
+  hostname: string
+  isBlockedListOpen: boolean
+  setBlockedListOpen: () => void
+  scriptsBlocked: number
+}
+
+interface State {
+  scriptsBlockedOpen: boolean
+  scriptsBlockedEnabled: boolean
+}
+
+export default class ScriptsControls extends React.PureComponent<Props, State> {
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      scriptsBlockedOpen: false,
+      scriptsBlockedEnabled: true
+    }
+  }
+
+  get tabIndex () {
+    const { isBlockedListOpen } = this.props
+    return isBlockedListOpen ? -1 : 0
+  }
```

Comment:
From my testing, I think we might want to forego the tabIndex functionality for now. It's awesome we have it and the focus box looks great, but I think we don't have the best parts of tabIndex wired up yet (like pressing Enter on keyboard to submit or change the toggle and then remove focus) so we are surfacing the orange box focus ring after any interactivity with toggles but then it doesn't go away and makes the component look like it's still active even though it's not. I'd vote for having this be inherited functionality when we consolidate Brave-UI components.

Reply:
hey Ross thanks for the feedback. I'm not sure I'm following, we need the  `tabindex`  so the control row (includes the arrow down, stats, and text) could be tabbable so using the space bar would trigger the blocked resources list, as expected from an a11y perspective. 

the toggle is independent of that and has `focus` on click just because that's how the outline works, but we're talking about a default behavior for the `input[type="checkbox"]` element.

the first alternative I proposed was to have `focus` only in the arrow down icon, but Karen said that it's better to have in the row as it is now. if you think this is wrong, I can change it back or we can figure out another way, but I guess we'll need Karen's input as well